docs/recipes/install/rocky9/x86_64/openchami/slurm/recipe.sh

```diff
@@ -0,0 +1,602 @@
+#!/bin/bash
+# -----------------------------------------------------------------------------------------
+#  Example Installation Script Template
+#  This convenience script encapsulates command-line instructions highlighted in
+#  an OpenHPC Install Guide that can be used as a starting point to perform a local
+#  cluster install beginning with bare-metal. Necessary inputs that describe local
+#  hardware characteristics, desired network settings, and other customizations
+#  are controlled via a companion input file that is used to initialize variables
+#  within this script.
+#  Please see the OpenHPC Install Guide(s) for more information regarding the
+#  procedure. Note that the section numbering included in this script refers to
+#  corresponding sections from the companion install guide.
+# -----------------------------------------------------------------------------------------
+
+inputFile=${OHPC_INPUT_LOCAL:-/opt/ohpc/pub/doc/recipes/rocky9/input.local}
+
+if [ ! -e ${inputFile} ];then
+   echo "Error: Unable to access local input file -> ${inputFile}"
+   exit 1
+else
+   . ${inputFile} || { echo "Error sourcing ${inputFile}"; exit 1; }
+fi
+
+# ---------------------------- Begin OpenHPC Recipe ---------------------------------------
+# Commands below are extracted from an OpenHPC install guide recipe and are intended for
+# execution on the master SMS host.
+# -----------------------------------------------------------------------------------------
+
+# Disable firewall
+systemctl disable firewalld
+systemctl stop firewalld
+
+
+# ------------------------------------------
+# Enable OpenCHAMI Repositories and Support 
+# ------------------------------------------
+dnf -y install podman buildah ansible-core nfs-utils tcpdump tftp curl
+rpm --import https://github.com/OpenCHAMI/release/releases/download/v0.0.30/public_gpg_key.asc
+dnf -y install https://github.com/OpenCHAMI/release/releases/download/v0.0.30/openchami-0.0.30.rpm
+
+
+# ----------------------------
+# OpenCHAMI No Proxy Settings 
+# ----------------------------
+export dns_name=$(hostname)
+export additonal_no_proxy="${dns_name},opaal,opaal-idp,smd,smd-init,hydra,cloud-init,acme-deploy,bss,haproxy,postgres,step-ca,configurator"
+if [ -z "$no_proxy" ]; then
+    export no_proxy="$additonal_no_proxy"
+else
+    export no_proxy="$no_proxy,$additonal_no_proxy"
+fi
+if [ -z "$NO_PROXY" ]; then
+    export NO_PROXY="$additonal_no_proxy"
+else
+    export NO_PROXY="$NO_PROXY,$additonal_no_proxy"
+fi
+
+
+# --------------------------------
+# Creating OpenCHAMI certificates 
+# --------------------------------
+echo "${sms_ip} ${dns_name}" | tee -a /etc/hosts > /dev/null
```

Comment:
This seems unnecessary complicated. I understand why this is done this way (probably because or originally running with sudo), but `echo "${sms_ip} ${dns_name}" >> /etc/hosts` should be enough here.